testsuite/tests/typing-modes/forkable.ml

```diff
@@ -254,11 +340,14 @@ type w1 : value mod global unforkable
 
 type w2 : value mod global forkable
 
+type w3 : value mod global spawnable
+
 [%%expect{|
 type ('a : value mod global) u1
 type ('a : value mod global unforkable) u2
 type w1 : value mod global unforkable
-type w2 : value mod global
+type w2 : value mod global forkable forkable
```

Comment:
@dkalinichenko-js I think #4931 isn't handling middle modes on an implied axis correctly, can you take a look?